msteams-platform/bots/how-to/conversations/channel-messages-with-rsc.md

```diff
@@ -41,12 +41,12 @@ The following code provides an example of the app manifest:
 
 ```json
 "webApplicationInfo": {
-"id": "XXxxXXXXX-XxXX-xXXX-XXxx-XXXXXXXxxxXX",
-"resource": "https://AnyString",
-"applicationPermissions": [
-"ChannelMessage.Read.Group"
-    ]
-  }
+  "id": "XXxxXXXXX-XxXX-xXXX-XXxx-XXXXXXXxxxXX",
+  "resource": "https://AnyString",
+  "applicationPermissions": [
+    "ChannelMessage.Read.Group"
+  ]
+}
```

Comment:
There are two misses here:
1. At line number 43 a parenthesis is missing.
2. Similarly, there is a closing parenthesis missing after line 49.

Sharing the screenshot of the correct snippet for your reference:

![image](https://user-images.githubusercontent.com/93579821/183596164-4a886048-01d3-4a9b-a091-39b5600e9f99.png)

Request you to kindly update json snippet.

Reply:
Hi @v-ypalikila, those parenthesis are left out (and also were not present before this change) because this is a sample snippet of a larger json document. See this screenshot from the page being modified which has a box around the snippet as part of the larger document

![image](https://user-images.githubusercontent.com/2782094/184040756-bbe1ba5c-154c-4919-802a-317a572ab8f8.png)

Reply:
Thank you @sfritz.